src/js/exporters.js

```diff
@@ -272,6 +276,57 @@ function exportKontaktScript() {
   return true
 }
 
+function exportImageLinePitchMap(range) {
+  if (exportError()) {
+    return
+  }
+  const NB_NOTES = 121 // IL products can only retune from C0 to C10
+  const HEADER_BYTES = Uint8Array.from([3, 0, 0, 0, 3, 0, 0, 0, NB_NOTES, 0, 0, 0])
+  const ENDING_BYTES = Uint8Array.from([0, 0, 0, 0, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255])
+
+  const tuningTable = model.get('tuning table')
+  const baseFreqOffset = Math.log2(tuningTable.baseFrequency / 440) // in number of octaves
+
+  // construct point data
+  let points = new ArrayBuffer(121 * 24)
+  let pointsDoubles = new Float64Array(points)
+  let pointsUint32 = new Uint32Array(points)
+  for (let i = 0; i < NB_NOTES; i++) {
+    const edo12cents = (i - 69) * 100
+    const offset = tuningTable.cents[i] - edo12cents
+    const normalizedOffset = ((offset / 1200 + baseFreqOffset) / range) * 0.5 + 0.5
+    const yCoord = Math.max(0.0, Math.min(1.0, normalizedOffset))
+    pointsDoubles[i * 3 + 1] = yCoord
+    if (i !== 0) { // no x offset and no curve data for first point
+      pointsDoubles[i * 3] = 0.0083333337679505 // constant x offset from previous point
```

Comment:
I suggest moving that fraction out to a constant similarly to `HEADER_BYTES` and `ENDING_BYTES`. It would decrease the "magic number" factor by giving context to it and would inform people where this number is coming from. Also, if this value can be calculated via some math, then I would definitely prefer that compared to having an arbitrary length number.